setup.py

```diff
@@ -74,13 +74,8 @@
     # your project is installed. For an analysis of "install_requires" vs pip's
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
-    install_requires=['hyperspy>=1.4', 'traitsui>=6.0'],
-
-    # List additional groups of dependencies here (e.g. development
-    # dependencies). You can install these using the following syntax,
-    # for example:
-    # $ pip install -e .[dev,test]
-    # extras_require={
-    #     'test': ['pytest'],
-    # },
-)
+    install_requires=['hyperspy>=1.5.dev', 'traitsui>=6.0'],
```

Comment:
Same as in https://github.com/hyperspy/hyperspy_gui_ipywidgets/pull/19#discussion_r307686020?